data/_ui/help.txt

```diff
@@ -152,27 +153,39 @@ help "outfitter"
 	`Here, you can buy new equipment for your ship. Your ship has a limited amount of "outfit space," and most outfits use up some of that space.`
 	`Some types of outfits have other requirements as well. For example, only some of your outfit space can be used for engines or weapons; this is your ship's "engine capacity" and "weapon capacity." Guns and missile launchers also require a free "gun port," and turrets require a free "turret mount." Also, missiles can only be bought if you have the right launcher installed.`
 	`Use your scroll wheel or click and drag, to scroll the view.`
-	`As in the trading and shipyard panels, you can hold down Shift, Control, or Alt to buy 5, 20, or 500 of an outfit at once, or multiple keys for larger amounts.`
+	`As in the trading and shipyard panels, you can hold down Shift, Control, or Alt to buy, sell or move 5, 20, or 500 of an outfit at once, or multiple keys for larger amounts.`
 	`You can also press F to search the outfitter for a specific item.`
 
 help "shipyard"
 	`Here, you can buy new ships.`
 	`Use your scroll wheel or click and drag, to scroll the view.`
 	`As in the trading and outfitter panels, you can hold down Shift, Control, or Alt to buy 5, 20, or 500 ships at once, or multiple keys for larger amounts.`
+	`You can use the shortcuts B to Buy, S to Sell the ship and outfits and R to sell the ship and retain the outfits if the planet has an outfitter.`
 	`You can also press F to search the shipyard for a specific ship.`
 
 help "cargo management"
-	`Sometimes you may want to buy some outfits directly into your cargo hold instead of having them installed. Perhaps a mission has asked you to retrieve some outfits or you want to carry extra ammunition to refill your ships somewhere that does not stock those weapons. To accomplish this, you need to switch out of installation mode into 'purchase to cargo' mode. This can be achieved by Control+clicking on your selected ship to deselect it.`
-	`When you are in 'purchase to cargo' mode, your ship information on the right-hand side will be replaced by information on how much cargo space you currently have available.`
-	`Outfits that are in your cargo hold can be (re)installed at any outfitter without cost.`
+	`Sometimes you may want to buy outfits directly into your cargo hold instead of having them installed. Perhaps a mission has asked you to retrieve some outfits or perhaps you want to carry extra ammunition to refill your ships somewhere that does not stock those weapons.`
```

Comment:
```suggestion
	`Sometimes you may want to buy outfits directly into your cargo hold instead of having them installed. Perhaps a mission has asked you to retrieve some outfits, or perhaps you want to carry extra ammunition to refill your ships somewhere that does not stock those weapons.`
```